stable/rabbitmq/README.md

```diff
@@ -77,10 +77,18 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `rabbitmq.configuration`             | Required cluster configuration                   | See values.yaml                                         |
 | `rabbitmq.extraConfiguration`        | Extra configuration to add to rabbitmq.conf      | See values.yaml                                         |
 | `rabbitmq.advancedConfiguration`     | Extra configuration (in classic format) to add to advanced.config    | See values.yaml                                         |
+| `rabbitmq.ssl.enabled`                  | Enable ssl support to rabbitmq |                     |
+| `rabbitmq.ssl.fail_if_no_peer_cert`     | `true`  | When set to true, TLS connection will be rejected if client fails to provide a certificate |
```

Comment:
To be consistent with the rest of the variables, can you use camelCase in the variable names?

Reply:
Is a mistake